portfolio/src/main/webapp/script.js

```diff
@@ -51,10 +51,14 @@ const form = document.getElementById("max-comments-form");
 form.addEventListener("submit", (e) => {
   e.preventDefault();
   const data = new FormData(form);
-  const newMaxComments = parseInt(data.get("max-comments"));
-  if (newMaxComments !== maxComments && newMaxComments !== NaN) {
-    maxComments = parseInt(newMaxComments);
-    getComments(maxComments);
+  for (const [key, val] of data) {
```

Comment:
might need a rebase or something?

Reply:
Hmm... When I run `git rebase master`, it says the current branch is up to date, even though this code is definitely out of date. Not sure what to do.

Reply:
Ok, rebased. This should be gone.